jsonld.js/src/test/scala/org/w3/banana/jsonldjs/io/JsonLdJsParserTest.scala

```diff
@@ -0,0 +1,39 @@
+package org.w3.banana
+package jsonldjs
+package io
+
+import zcheck.SpecLite
+import scalajs.concurrent.JSExecutionContext.Implicits.runNow
+import org.w3.banana.plantain._
+
+object JsonLdJsParserTest extends SpecLite {
```

Comment:
Good that you have ported the test, one more of the list:) Re the missing writer.  Perhaps we should split SerialisationTestSuite into  reader/writer tests?